docs/dev_install.rst

```diff
@@ -22,18 +22,20 @@ Installing Depedencies
     python3 -m pip install --user setuptools astpretty astor pytest opencv-python pybind11
     python3 -m pip install --user Pillow numpy scipy GitPython yapf colorama psutil autograd
 
-* (If on Ubuntu) Execute ``sudo apt install libtinfo-dev clang-8`` (or ``clang-7`` should work as well).
+* (If on Ubuntu) Execute ``sudo apt install libtinfo-dev clang-8``.
```

Comment:
I found `clang-7` not working well with LLVM 8 (#866).

Reply:
I think it's caused by https://github.com/taichi-dev/taichi/pull/874. I've been using `clang-7` with no issues.

Reply:
Thank for the hard work! I'll try #866 again later and see if the issue is gone.